doc/dev/getting-started.rst

```diff
@@ -63,176 +63,102 @@ One-time Preparation
 
 #. Install the dependencies.
 
-   OS X users
+   i. Mac users
 
-   a.  Install Xcode from the `Mac App Store (requires root)
-       <https://developer.apple.com/xcode/>`_.
-   #.  `Register as an Apple Developer
-       <https://developer.apple.com/register>`__.
-   #.  Install the Xcode command-line tools: Xcode -> preferences ->
-       Downloads -> Command Line Tools (requires root).
+      a.  Install Xcode with::
 
-   Linux users
+             xcode-select --install
 
-   a.  Install the python development environment, virtualenv, pip, gcc, and
-       g++.
+      #.  Install `Homebrew <http://brew.sh/>`__ with::
 
-       On recent Debian and Ubuntu this can be done with::
+             /usr/bin/ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"
 
-           sudo apt-get install python2.7-dev python-virtualenv python-pip gcc \
-           g++ git astyle gcovr cppcheck enchant
+      #.  Install the python development environment and some additional development packages::
 
-       For RHEL6::
+             brew install python astyle gcovr cppcheck enchant
+             sudo pip install --upgrade pip virtualenv
 
-           sudo yum install -y python-devel python-pip git gcc gcc-c++ make enchant
-           sudo pip install virtualenv
+   ii. Linux users
```

Comment:
Luckily it shows up fine when I `make doc` locally.

<img width="400" alt="screen shot 2016-11-16 at 10 02 59 am" src="https://cloud.githubusercontent.com/assets/566823/20359255/ed386516-abe3-11e6-9771-36c2d44087dc.png">

That said, we don't need a single sub-bullet point, so I'll fix that anyway. Maybe that will fix GitHub's rendering that section as a blockquote.